R/countries.R

```diff
@@ -122,9 +130,11 @@ getBenefitsTable <- function(countryData, max=1000, grid=1) {
 #' @importFrom readxl read_excel
 #' @import data.table
 #' @importFrom stats weighted.mean
-#' 
+#' @examples
+#' countrydata <- loadCountryData('extdata/countryData.xlsx')
 loadCountryData <- function(filename, Gavi=F) {
-  rawData <- data.table(read_excel("../../Other/highrisk_clean_bt.xlsx"))
+
+  rawData <- data.table(read_excel(system.file(filename,package = 'vaccineEarlyInvest')))
```

Comment:
This seems a bit wrong. In all functions that use `system.file()` you should have two options
1) using `system.file()` as a default
2) using your own csv or Excel files

That means you want this:

```
loadCountryData <- function(filename, Gavi=F) {
  rawData <- data.table(read_excel(filename))
```

But when calling `loadCountryData()` you sometimes will do `loadCountryData(system.file("extdata", "countryData.xlsx" ,package = 'vaccineEarlyInvest')))`
And sometimes you will do `loadCountryData("c:/somefolder/some_file.xlsx")`.

You were testing for the latter previously when you wrote "../inst/" filepath! So you can now easily test for both.

Reply:
I see what you mean